source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -190,16 +190,31 @@ HTTP Webhook Log Target
       This setting corresponds to the
       :envvar:`MINIO_LOGGER_WEBHOOK_PROXY` environment variable.
 
+   .. mc-conf:: queue_dir
+
+      .. versionadded:: RELEASE.2023-05-18T00-05-36Z
+
+      *Optional*
+
+      Specify the directory path to enable MinIO's persistent event store for undelivered messages, such as ``/home/events``.
```

Comment:
I wonder if a user would read 'such as...' as being related to the message itself. Maybe

```suggestion
Specify the directory path, such as ``/opt/minio/events``, to enable MinIO's persistent event store for undelivered messages.
```

Also suggest ``/opt/minio`` or something similar. ``/home`` might not work since we usually run as a system service.

We might need to add a line "The MinIO process must have read/write/list access on the specified directory" since not everyone thinks about permissions.

Reply:
If `home/events` is a bad choice, should the other instances of that get updated also? I got that text from a similar queue path setting.

Reply:
Added your reword manually 👍🏻

Reply:
Also replaced other occurrences of `/home/events`